.pre-commit-config.yaml

```diff
@@ -1,21 +1,21 @@
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
+  - repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
I cleaned up the indenting on this yaml file a bit as I found it really odd to try and read

I've made all the whitespace only changes in one commit and functional changes in another.